sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -51,8 +51,8 @@ static void applyMetadata(Context context, SentryAndroidOptions options) {
             .log(SentryLevel.DEBUG, "isAnrReportInDebug read: %s", isAnrReportInDebug);
         options.setAnrReportInDebug(isAnrReportInDebug);
 
-        int anrTimeoutIntervalMills =
-            metadata.getInt(ANR_TIMEOUT_INTERVAL_MILLS, options.getAnrTimeoutIntervalMills());
+        long anrTimeoutIntervalMills =
+            metadata.getLong(ANR_TIMEOUT_INTERVAL_MILLS, options.getAnrTimeoutIntervalMills());
```

Comment:
Identified while writing docs, Mills should be always a long.
it could be a separated PR, yes, but I was lazy to do it :(